day8/alienalphabet.py

```diff
@@ -0,0 +1,36 @@
+class Node:
+    def __init__(self, val):
+        self.val = val
+        self.next = None
+
+def linearize(DAG):
+    sourceNodes = sources(DAG)
+    visited = set()
+
+    def dfs(node, visited, postOrderStack):
+        if node.next == None:
+            # sink node
+            postOrderStack.append(node)
+        dfs(node.next)
+
+    for node in sourceNodes:
+        # run DFS to get post order numbers
+        # don't need the actual numbers, just put the node into a stack
+        # when you're about to label the post number
```

Comment:
oh this file shouldn't be in here, it's in a different PR I haven't merged yet. Thanks for the catch! I'll remove it.